### DISCUSSION TITLE
Bowtie2 not found, environment management with pipenv

### DISCUSSION THREAD

**Discussion by MAIA-FPC** [Kinggerm/GetOrganelle#190] [Q&A]
Hi I'm repeatedly encountering an issue with the Bowtie2 dependency when running `get_organelle_config.py --add embplant_pt,embplant_mt` 

I'm running Monterey 12.5
python 3.7.12
GetOrganelle 1.7.4.1

I followed Installation instructions from Option 2 replacing pip with pipenv to install into my venv. When reaching the step above (`get_organelle_config.py`) I get the error `ERROR: Bowtie2 is not available!`

This issue aligns with user reported issues in conda (#102 , #85, #129 ). I attempted to install TBB version 2020.2 as suggested in those threads but the only available whl for tbb for macOS on PYPI is 2019.0 or 2020.3.254.

Does anyone have a whl compatible with GetOrganelle and macOSX? Or is my best approach to ditch pipenv and start a conda venv.

**Comment by Kinggerm** [comment]
Hi, thanks for reporting this issue. I have not the same environment to test yet.

Have you tried the provided binaries or any other approaches to install bowtie2 independently?
